src/views/Pools/index.tsx

```diff
@@ -61,6 +67,7 @@ const Pools: React.FC = () => {
         <FlexLayout>
           <Route exact path={`${path}`}>
             <>
+              <CakeVaultCard pool={cakePool} account={account} />
```

Comment:
As discussed - does not get filtered by StakedOnly toggle

Reply:
Too important. Ahaha. I've added it to the Trello 👌